schemas/string_interpolation/transforms/generate_date_range.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/generate_date_range.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "generate_date_range": {
+    "description": "A transform for generating the start and end date of a date range from a reference date, offset number of weeks, number of days in range and an optional first day of the week.",
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["generate_date_range", "format_date_range_pair"]
```

Comment:
How come both transforms are defined here?.

Reply:
yeah this is what tripped me up and I was [asking](https://surveydatacollection.slack.com/archives/CLR17KQKU/p1633009031014500) what this section does - I'm still not clear then so I'll catch up about it in a bit

Reply:
This is the literal "name" of the transform. `enum` was used in the old JSON draft, since the latest draft `enum` could be dropped for `const` when we only have one value, so instead of enum, it would be like `"const": "format_date_range"` etc.